src/Client/Updater.php

```diff
@@ -575,4 +525,70 @@ private function getUpdateStartTime(): \DateTimeImmutable
     {
         return (new \DateTimeImmutable())->setTimestamp($this->clock->getCurrentTime());
     }
+
+    /**
+     * Performs a delegation search for a target metadata object.
+     *
+     * @param \Tuf\Metadata\TargetsMetadata|null $targetsMetadata
+     *   The targets metadata to search.
+     * @param string $target
+     *   The path of the target file. Needs to be known to the most recent
+     *   targets metadata downloaded in ::refresh().
+     * @param string[] $searchedRoles
+     *   The roles that have already been searched. This is for internal use only and should not be passed by calling code.
+     *   calls to this function and should be provided by any callers.
+     * @param bool $terminated
```

Comment:
No description of this one? At the very least it should say that it's for internal use only.

Reply:
added description